src/gatling/resources/RecordedSimulationFRApplicant_0044_request.txt

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
is this file needed? also that event_token is dynamic and will expire after a few hours

Reply:
Yes, all files are needed. Made token dynamic.

Reply:
Yes, the content of the file is needed. Made token dynamic.